.github/workflows/android_clean_build.yml

```diff
@@ -0,0 +1,105 @@
+name: Android clean build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ "main" ]
+  push:
+    tags:
+      - '*'
```

Comment:
Does this only run a gob is a tag is present?

Reply:
When a tag is created.